lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::IndegoBikeshareLocations;
```

Comment:
This needs to be `DDG::Spice::BikeSharing::IndegoBikeshareLocations` to utilize the BikeSharing namespace that @marianosimone created.

However, that's pretty lengthy. So I think something like `DDG::Spice::BikeSharing::Phl` (and similarly `DDG::Spice::BikeSharing::Nyc`) would be good.

Any thoughts on that @AcriCAA , @marianosimone? It seems like we're going to have several of these so some standard naming convention would be nice. Are there multiple bike sharing providers per city? If so then maybe something like `BikeSharing::PHLIndego`, `BikeSharing::NYCCiti`?